client/src/components/SideInfoPanel/SideInfoPanel.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import {
+  SidePanelImg,
+  ShortDescription,
+  Container,
+  ReviewInfo,
+  ReviewWrapper,
+  OverallReview,
+  ReleaseDateWrapper,
+  ReleaseInfo,
+  ReleaseDate,
+  PubAndDev,
+  TagsWrapper,
+  TagList,
+  Tag,
+  ToolTip,
+} from './styles';
+import GamesContext from '../../context';
+
+const SideInfoPanel = () => {
+  const { sidePanelImg, sidePanelInfo } = useContext(GamesContext);
+  console.log(sidePanelInfo);
+  return (
+    <Container>
+      <SidePanelImg src={sidePanelImg} />
+      <ShortDescription>{sidePanelInfo.short_description}</ShortDescription>
+      <ReviewWrapper>
+        <ReviewInfo>
+          Recent Reviews:
+        </ReviewInfo>
+        <OverallReview>
+          <ToolTip className="tooltip">review stats</ToolTip>
+          {sidePanelInfo.recent_reviews}
+        </OverallReview>
+        <ReviewInfo>
+          {`(${sidePanelInfo.recent_reviews_count})`}
+        </ReviewInfo>
+      </ReviewWrapper>
+      <ReviewWrapper>
+        <ReviewInfo>
+          All Reviews:
+        </ReviewInfo>
+        <OverallReview>
+          <ToolTip className="tooltip">review stats</ToolTip>
+          {sidePanelInfo.all_reviews}
+        </OverallReview>
+        <ReviewInfo>
+          {`(${sidePanelInfo.all_reviews_count})`}
+        </ReviewInfo>
+      </ReviewWrapper>
+      <ReleaseDateWrapper>
```

Comment:
I like that you're using a lot of wrappers here though it can seem like a pain to do so, I've found its much easier to change position/style of a bunch of smaller elements that are grouped together if I have them wrapped in a wrapper like you're using here so overall it makes life easier